testing/functional/tests/cypress/page-objects/student-objects/ApplicationObject.ts

```diff
@@ -193,15 +194,15 @@ export default class ApplicationObject {
   }
 
   myStudyPeriodIsNotListedCheckbox() {
-    return cy.get("#eoslne6-offeringnotListed");
+    return cy.get("#etvv9qs-programnotListed");
```

Comment:
Using the IDs for the form.io definitions can be tricky because they can be auto regenerated on not even be in sync for different program years, for instance. We should rely on items that are not likely to change like the _name_ that is also used in the backend and is not supposed to be changed or will be changed under controlled circumstances.

Reply:
I saw that this specific one was changed but I can see that the source code has more than 100 occurrences of a similar case.
Just keep in mind the dynamic nature of the form.io id generation in the future. Maybe we should rely as much as possible in the name instead of the ids or whatever else you think would be better.

Reply:
@bhavikccits, please consider taking a look on this also: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements